addons/mrp/i18n/mrp.pot

```diff
@@ -2742,6 +2742,7 @@ msgstr ""
 
 #. module: mrp
 #: model:ir.model.fields,field_description:mrp.field_mrp_production__location_src_id
+#: model:ir.model.fields,field_description:mrp.field_mrp_routing__location_id
```

Comment:
So after this PR we have 2 msgids marked as translations for the same field, and it breaks loading translations (in "overwrite" mode) due to the way `ON CONFLICT DO UPDATE` works.

I find this loading mechanism a bit brittle. How about we improve it like this:
- we add a test that parses all POTs and verifies that there is no duplication of source tag comments, to detect this conflict case immediately. That should not be difficult to write. (@moylop260 installing the translations is a good test idea too and we should do it, but detecting conflicts by parsing the POs is very direct and doesn't require a 100% complete translation)  
- and/or we change the `INSERT SELECT` in the [translation queries](https://github.com/odoo/odoo/blob/908f6b8b56323f27f9362dc5ad70ffcd313c4c4e/odoo/addons/base/models/ir_translation.py#L138-L182) into a `INSERT SELECT GROUP BY (...)` to merge all duplicates rows in the `SELECT`.

The second option is a bit redundant if we implement the first one, it's mostly an extra safeguard. And we'd need to verify that it does not impact the loading time for translations.

What do you think @nim-odoo ?